anesthetic/plot.py

```diff
@@ -697,6 +697,7 @@ def make_2d_axes(params, labels=None, lower=True, diagonal=True, upper=True,
                          ticks=ticks,
                          gridspec_kw=gridspec_kw,
                          subplot_spec=subplot_spec)
+    fig.align_labels()
     return fig, axes
```

Comment:
I snuck this little line in. I can revert it if so wished, but I find this significantly improves the appearance of triangle plots by aligning the labels independent of the length of the tick labels. Here a comparison:

* without alignment:
  ![image](https://github.com/handley-lab/anesthetic/assets/18258042/014f1edd-744c-4dd9-8c09-3380a6a8e939)
* with alignment:
  ![image](https://github.com/handley-lab/anesthetic/assets/18258042/b2d1c085-e71f-4659-89c7-4556a01d2fff)